src/main/java/entities/post.java

```diff
@@ -0,0 +1,114 @@
+package entities;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * The Post class contains all the necessary information of a post: its ID, title,
+ * description,tags,collaborators, deadline, and users who favourited it.
+ */
+public class Post extends Postable{
+    public static int numPostsCreated = 0;
+    /**The id is equal to the number of posts created.*/
+    private int id;
+    /**This is a list of the users who favourited this post.*/
+    private List<User> favouritedUsers;
+    private String title;
+    private List<String> tags;
+    private String collaborators;
```

Comment:
Should collaborators be a list of users?

Reply:
I think collaborators is a presented attribute, sort of like qualifications, etc.

Reply:
do we need collaborators?